pkg/booksapi/server.go

```diff
@@ -0,0 +1,26 @@
+package booksapi
+
+import (
+    "fmt"
+    "net/http"
+	"os"
+)
+
+func handler (w http.ResponseWriter, r *http.Request){
+	query, err := QueryTitle("salo", -1)
+	if err != nil {
+		fmt.Println("Query failed!", err)
+		os.Exit(1)
```

Comment:
maybe take this line out because it'd kill the server if it cant reach the openlibrary api

Reply:
lol fair